app/components/UI/Bridge/components/TokenButton.tsx

```diff
@@ -53,6 +53,9 @@ const createStyles = (params: StylesParams) => {
       color: theme.colors.text.default,
       fontSize: 24,
     },
+    networkBadge: {
+      borderColor: theme.colors.background.section,
```

Comment:
I think we want this to match the background color of the pillContainer

```suggestion
      borderColor: theme.colors.background.muted,
```